products/plesk.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Plesk
+category: server-app
+iconSlug: plesk
+permalink: /plesk
+versionCommand: plesk version
+releasePolicyLink: https://www.plesk.com/lifecycle-policy/
+eolColumn: End Of Life
+releaseColumn: true
+releaseDateColumn: true
+
+releases:
+-   releaseCycle: "18.0.50"
+    releaseDate: 2023-01-31
+    eol: 2023-04-23
+    latest: "18.0.50.2"
+    latestReleaseDate: 2023-02-14
+    link: https://docs.plesk.com/release-notes/obsidian/change-log/#plesk-18050
+
+-   releaseCycle: "18.0.49"
+    releaseDate: 2022-12-20
+    eol: 2023-03-12
+    latest: "18.0.49.2"
+    latestReleaseDate: 2023-01-11
+    link: https://docs.plesk.com/release-notes/obsidian/change-log/#plesk-18049
+
+-   releaseCycle: "17"
+    releaseDate: 2016-10-11
+    eol: 2021-04-20
+    latest: "17.8.11.95"
+    latestReleaseDate: 2021-01-25
+    link: https://docs.plesk.com/release-notes/onyx/change-log/
+---
+
+> [Plesk](https://www.plesk.com/) is a proprietary hosting control panel developed by WebPros
```

Comment:
A summary of the "Plesk Lifecycle Policy" page would be great. Something like :

> In the past (e.g. 12.x > 17.x), Plesk used to launch one major version per year. With Plesk 18.x the release cadence changed to one patch release a month, with each release supported for one two months...

Reply:
I prepared a recap of the policy. Hope it helps.

Reply:
Yep. I updated it a little bit, let me know if it looks good to you.

Reply:
Thank you, looks very good! 👍